### PR TITLE
Use app.ico as icon instead of app.png for setting window

### PR DESCRIPTION
**Pull request by taooceros**
It will help scaling, although seems that our icon is too complex to display well in 16*16 scale.

#86

### CODE REVIEW

File: Flow.Launcher/Flow.Launcher.csproj [hunk 1]
```diff
@@ -94,6 +97,12 @@
     <ProjectReference Include="..\Flow.Launcher.Plugin\Flow.Launcher.Plugin.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Images\app.ico">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
```
Comment (jjw24) [inline Flow.Launcher/Flow.Launcher.csproj:105]: this is not needed right since you are including all .ico extension

**Comment by jjw24** [comment]
Does this actually solve the problem in issue 86?

**Comment by taooceros** [comment]
> Does this actually solve the problem in issue 86? As in will it actually improve the rendering?

The rendering won't be changed, but the ico file contains 16*16 icon so wpf don't need to downscaling the icon.
Well, if the 16*16 icon satisfying, it can be solved, but actually it isn't not that perfect.

File: Flow.Launcher/Flow.Launcher.csproj (hunk 1 above)
Comment (taooceros) [reply Flow.Launcher/Flow.Launcher.csproj:105]: oh yes

**Comment by taooceros** [comment]
image
This is the setting panel icon after the pr.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**